<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DobotM1 demo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/resources/icon.png</normaloff>:/resources/resources/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buttonSearch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpPump">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Pump</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioPumpOn">
       <property name="text">
        <string>On</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioPumpOff">
       <property name="text">
        <string>Off</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="grpPumpDirection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Direction</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioPumpPush">
       <property name="text">
        <string>Push</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioPumpSuck">
       <property name="text">
        <string>Suck</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="buttonGoHome">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Home safe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCheckAlarm">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Alarms</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpCoordinates">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Joint / Cartesian</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="3">
      <widget class="QLabel" name="txtR">
       <property name="text">
        <string>R =</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="txtY">
       <property name="text">
        <string>Y =</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="txtZ">
       <property name="text">
        <string>Z =</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="txtX">
       <property name="text">
        <string>X = </string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="txtJ1">
       <property name="text">
        <string>J1 = </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="txtJ2">
       <property name="text">
        <string>J2 = </string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="txtJ3">
       <property name="text">
        <string>J3 =</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="txtJ4">
       <property name="text">
        <string>J4 =</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="buttonClearAlarms">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ClearAlarms</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPtpX">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPtpY">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>260</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPtpZ">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>290</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonExecPtpJ">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>350</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PTP MOVJ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPtpR">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>320</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClearQueue">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear tasks</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonGoHomeSimple">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>HomeSimple</string>
    </property>
   </widget>
   <widget class="QComboBox" name="listDobots">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>192.168.1.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>192.168.1.6</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="txtConnectedDobotName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dobot name</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtAlarms">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnXpJ1p">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnXmJ1m">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnYpJ2p">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Y+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnYmJ2m">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Y-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnZmJ3m">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>330</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Z-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRmJ4m">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>R-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnZpJ3p">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>270</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Z+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRpJ4p">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>R+</string>
    </property>
   </widget>
   <widget class="QComboBox" name="listJogMode">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>220</y>
      <width>72</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Axes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Joints</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="buttonExecPtpL">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>380</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PTP MOVL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>235</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>265</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>295</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>325</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>buttonSearch</tabstop>
  <tabstop>listDobots</tabstop>
  <tabstop>txtPtpX</tabstop>
  <tabstop>txtPtpY</tabstop>
  <tabstop>txtPtpZ</tabstop>
  <tabstop>txtPtpR</tabstop>
  <tabstop>buttonExecPtpJ</tabstop>
  <tabstop>radioPumpOn</tabstop>
  <tabstop>radioPumpOff</tabstop>
  <tabstop>radioPumpPush</tabstop>
  <tabstop>radioPumpSuck</tabstop>
  <tabstop>buttonGoHome</tabstop>
  <tabstop>buttonGoHomeSimple</tabstop>
  <tabstop>buttonCheckAlarm</tabstop>
  <tabstop>buttonClearAlarms</tabstop>
  <tabstop>buttonClearQueue</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
